climada/hazard/centroids/centr.py

```diff
@@ -928,30 +928,42 @@ def set_elevation(self, topo_path):
             self.set_meta_to_lat_lon()
         self.elevation = u_coord.read_raster_sample(topo_path, self.lat, self.lon)
 
-    def set_dist_coast(self, signed=False, precomputed=False, scheduler=None):
+    def set_dist_coast(self, signed=False, precomputed=True, scheduler=None):
         """Set dist_coast attribute for every pixel or point in meters.
 
+        The distances are read from a raster file containing precomputed distances (from NASA) at
+        0.01 degree (approximately 1 km) resolution.
+
         Parameters
         ----------
         signed : bool
-            If True, use signed distances (positive off shore and negative on land). Default: False.
-        precomputed : bool
-            If True, use precomputed distances (from NASA). Default: False.
-        scheduler : str
-            Used for dask map_partitions. "threads", "synchronous" or "processes"
+            If True, use signed distances (positive off shore and negative on land). Default: False
+        precomputed : bool, optional
+            Whether distances should be read from a pre-computed raster (True) or computed
+            on-the-fly (False). Default: True.
+            
+            .. deprecated:: 4.1
+               Argument is ignored, because distances are not computed on-the-fly anymore.
+ 
+        scheduler : str, optional
+            Scheduler for on-the-fly computation.
+            
```

Comment:
```suggestion

            .. deprecated:: 4.1
               Argument is ignored, because distances are not computed on-the-fly anymore.

        scheduler : str, optional
            Scheduler for on-the-fly computation.

```